### ISSUE TITLE
Game does not load if thaumcraft not installed

### ISSUE THREAD

**Issue by mezz** [MagicBees/MagicBees#24]
Game crashes prior to main menu if Thaumcraft is not installed. The crash is related to the Thaumium Scoop. This crash occurs even if the Thaumcraft module is disabled in the config. See attached error log.

https://gist.github.com/mezz/ce0ab009e984da1806f0

**Comment by mezz** [comment]
_From [Arkandos](http://bitbucket.org/Arkandos) on 2014-11-30 01:47:29+00:00_

Please, whenever you post a bugreport, atleast give me the Magic bees and thaumcraft version you are using.

**Comment by mezz** [comment]
_From [omelet](http://bitbucket.org/omelet) on 2014-11-30 17:46:52+00:00_

My bad. Using magicbees-1.7.10-2.1.22, and Thaumcraft is not installed.

**Comment by mezz** [comment]
_From [Arkandos](http://bitbucket.org/Arkandos) on 2014-12-01 17:08:11+00:00_

Interesting problem. If you add thaumcraft, does it resolve itself?

**Comment by mezz** [comment]
_From [omelet](http://bitbucket.org/omelet) on 2014-12-01 23:35:09+00:00_

It does resolve the issue if I install Thaumcraft.

I usually use JRE 1.8.0_11 x64, so I thought that might be the issue, but after trying with 1.7.0_71 x64 and 1.7.0_67 x86, I'm getting the same issue with each of these versions as well.

Looking around the source, I thought it might be an issue with the "Thaumcraft" in the Optional.Interface section being uppercase, vs. the lowercase used in "thaumcraft.api.IRepairable", so I searched around for other mods that have something from Thaumcraft as an Optional.Interface. What I found is that they all capitalized Thaumcraft in the modid just like you did, but they also all included the full class path (not sure if I'm using the right words there) in the iface (i.e. they would have set iface="thaumcraft.api.IRepairable", whereas you just have iface="IRepairable"). Could that be the problem, that Forge isn't removing the reference to thaumcraft.api.IRepairable because the whole thing isn't written out in the Optional.InterfaceList?

edit: removed @ symbols because they turn into weird formatting.
